### PR TITLE
Subset sntemp psu

### PR DESCRIPTION
**Pull request by jzwart**
I `gd_put( )` the SNTemp input and output  for each segment in `3_predictions/in` and then subset those input/output using unique set of seg_id_nat from `9_collaborator_data/psu/high_obs_upstream_sites.csv`. I've also added the variable description text file to the PSU folder. 

Not sure what file format they want it in, but right now it is in `.feather`.

### CODE REVIEW

**Comment by jzwart** [comment]
And sorry, a lot of the changes are from my R getting rid of line endings

**Comment by aappling-usgs** [comment]
FYI you can hide whitespace changes when reviewing PRs through this setting:
image

**Comment by jzwart** [comment]
Ah, thanks @aappling-usgs !

**Comment by jzwart** [comment]
There are now 220 unique `seg_id_nat` in the subset sites for PSU. Does that sound right?

**Comment by limnoliver** [comment]
Yep, that number sounds right. Thanks for this, Jake!

File: 9_collaborator_data/psu/model_variable_description.txt [hunk 1]
```diff
@@ -0,0 +1,198 @@
+Description of variables and units. 2019-09-30
```
Comment (jsadler2) [inline 9_collaborator_data/psu/model_variable_description.txt]: This file is great to have!

**Pull request merged**